bazel/bazel.sh

```diff
@@ -1,20 +1,14 @@
 #!/bin/bash
 
-echo
-echo           \*\*\*\*\* DEPRECATION NOTICE \*\*\*\*\*
-echo
-echo This image is deprecated and will no longer be updated.
-echo This recent version of the image will continue to exist.
-echo
-echo In place of this image, please use one of the official
-echo \`bazel\` images available at
-echo \`gcr.io/cloud-marketplace-containers/google/bazel\`.
-echo
-echo For details, visit
-echo http://gcr.io/cloud-marketplace-containers/google/bazel
-echo
-echo           \*\*\*\*\* DEPRECATION NOTICE \*\*\*\*\*
-echo
+echo '
+                   ***** NOTICE *****
+
+A supported `bazel` image, including multiple tagged versions,
```

Comment:
Same as above

Reply:
Same reply -- IDK if it's "the" or if there are others.